src/dotnet/APIView/APIView/CodeFileRenderer.cs

```diff
@@ -21,11 +21,11 @@ public RenderResult Render(CodeFile file, bool showDocumentation = false, bool e
             return new RenderResult(codeLines.ToArray(), sections);
         }
 
-        public CodeLine[] Render(CodeFileToken[] tokens)
+        public CodeLine[] Render(CodeFileToken[] tokens, bool showDocumentation = false, bool enableSkipDiff = false)
```

Comment:
this change is already in main https://github.com/Azure/azure-sdk-tools/blob/main/src/dotnet/APIView/APIView/CodeFileRenderer.cs#L16 looks like there is an issue with your rebase

Reply:
There are two methods though.